lexers/s/sketchdrawing.go

```diff
@@ -0,0 +1,18 @@
+package s
+
+import (
+	. "github.com/alecthomas/chroma" // nolint
+	"github.com/alecthomas/chroma/lexers/internal"
+)
+
+// SketchDrawing lexer.
+var SketchDrawing = internal.Register(MustNewLexer(
```

Comment:
Didn't find it on pygments nor Chroma.

Reply:
This is a wakatime top language: https://wakatime.com/leaders/languages.